### PR TITLE
Add Eniac's BID and Translate in Chiniese

### PR DESCRIPTION
**Pull request by EniacTNB**
- add eniac to bid code project 
- translate README 、CODE_OF_CONDUCT、CONTRIBUTION-GUIDELINES.md、LICENSE、about.md

### CODE REVIEW

**Comment by XanderRubio** [comment]
Hi @EniacTNB! Thank you for your time and contribution. I'm currently traveling and will be reviewing as soon as I'm settled into a workspace. Have a great day and I look forward to reviewing and merging your co contribution. Thank you for your patience. 

Xander

File: src/Jsons/Contributors.json
Comment (XanderRubio) [inline src/Jsons/Contributors.json:1]: I thought I was able to display your numbered list on new lines but I haven't been able to figure out in the value "text". Looks great!

**Review by XanderRubio — APPROVED** [review]
Your contributions are greatly appreciated, and thank you for taking the time to translate the docs, Zhang! Living in a place surrounded by mountains, snow, and water can indeed be magical, providing clarity and peace when immersed in such natural beauty. Your second point emphasizes living in the moment and being present with your loved ones and family. By taking the time to write this and directing your life course towards your numbered item list. I appreciate and am grateful for your contribution, assistance, and for sharing your text and image of what you want to do before you die. Clapping Hands

**Comment by XanderRubio** [comment]
Thank you, @EniacTNB! I am pleased to inform you that your pull request has been merged successfully✅! You have done an excellent job, and I congratulate you on this achievement. 🎉

Furthermore, I invite you to check out the new [live link](https://before-i-die-achievements.vercel.app/), where you will notice changes in the user interface since you last worked on your version. Your continued support and contributions to improving Before I Die Code are greatly appreciated. Thank you for your efforts in enhancing the documentation for translation and assisting new developers from China when they first encounter this project.

I am happy to report that this pull request has successfully resolved issue #32.

Have a great week!

Xander

**Comment by EniacTNB** [comment]
Hey, Clemens! @XanderRubio 

Thanks to the BID Code Project, it's the first time I've really thought about what is truly important to me. When I opened my photo album, hoping to find a picture that could represent my BID dream, I couldn't find one. It made me realize that I should dedicate more time to my goals.

I also wanted to express my gratitude for approving my pull request. It was my first PR, and I'm thrilled to have contributed to the project.

Wishing you a fantastic journey!

Best regards
ZZ

**Comment by XanderRubio** [comment]
> Hey, Clemens! @XanderRubio
> 
> Thanks to the BID Code Project, it's the first time I've really thought about what is truly important to me. When I opened my photo album, hoping to find a picture that could represent my BID dream, I couldn't find one. It made me realize that I should dedicate more time to my goals.
> 
> I also wanted to express my gratitude for approving my pull request. It was my first PR, and I'm thrilled to have contributed to the project.
> 
> Wishing you a fantastic journey!
> 
> Best regards ZZ

Thanks for replying @EniacTNB! It is my pleasure as the main goal and value of this repo is to help new developers get involved in Before I Die Code. Additionally is an open source project that developers can contribute to the growth of how the project grows over time from UI, tech stack, and adding ML/AI into the project. I'm happy to hear that you spent time dedicated to defining what is important to you in your life. It is a valuable exercise that creates ripple effects in one's life. I'm happy to hear you had a realization, and this project has helped with contributing to your first PR and also helping to dedicate more time to your goals. You are more than welcome to continually contribute and apply your knowledge and expertise to help grow Before I Die Code! Thank you for contributing and have a great week! 

Please [connect with me on LinkedIn](https://www.linkedin.com/in/alexanderclemens/) as I would like to thank each contributor every week for this repo in video format and tag them on LinkedIn.

**Pull request merged**